src/Chapter4.hs

```diff
@@ -629,7 +636,7 @@ Can you implement a monad version of AND, polymorphic over any monad?
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
 andM :: (Monad m) => m Bool -> m Bool -> m Bool
-andM = error "andM: Not implemented!"
+andM ma mb = (\a -> if a then  mb else return a) =<< ma
```

Comment:
Nice lazy implementation 🙂 
`>>=` is much more canonical, and in here `=<<` doesn't seem to bring any improvements in this particular case. With `>>=` you could also omit parenthesis 🙂